client/src/ledger_app_clients/ethereum/eip712/InputData.py

```diff
@@ -17,6 +17,7 @@
 # global variables
 app_client: EthAppClient = None
 filtering_paths: dict = {}
+filtering_tokens: list[dict] = list()
```

Comment:
Minor, but initializing with simply `[]` instead of `list()` seems more python oriented.